libs/bower/README.md

```diff
@@ -0,0 +1,24 @@
+# How to add dependencies with bower in ILIAS
+
+**New dependencies MUST be approved by the Jour Fixe of the ILIAS society or by the Technical Board.**
```

Comment:
Does the TB want to approve bower dependencies? Or is this for cases where the JF forwards decisions to the TB? I would still prefer to have a JF date in the metadata.